index.html

```diff
@@ -406,120 +436,129 @@
+                <h4 class="text--subheading">Algorithmic Content</h4>
+                <p class="diagram-detail__card__content">Generating believable messages that don’t feel artificial or templated is key to maintaining the illusion. Tools like Automated Insights Wordsmith hint to possibilities where content can be produced on the fly and personalized to each user.</p>
               </div>
             </div>
 
-            <div class="diagram-details">
-              <div class="diagram-detail">
+            <div class="diagram-detail">
+              <div class="diagram-detail__card">
                 <!--transparent borders-->
                 <div class="icon -borders"></div>
                 <!--background-color-->
                 <div class="icon -color">
                   <!--img-->
-                  <img  class="resize" src="./public/img/icon-processing.png">
+                  <img  class="resize" src="./public/img/icon-processing.png" alt="">
                 </div>
-                <p class="text -small -color-accent -caps -bold centered">Natural Language Processing</p>
-                <p class="text -small content">Florence’s conversational style will encourage users to speak to her like a friend. Natural language processing, like those offered by IBM’s Watson, will be needed to extract meaning and respond intelligently.</p>
+                <h4 class="text--subheading">Natural Language Processing</h4>
+                <p class="diagram-detail__card__content">Florence’s conversational style will encourage users to speak to her like a friend. Natural language processing, like those offered by IBM’s Watson, will be needed to extract meaning and respond intelligently.</p>
               </div>
             </div>
 
-            <div class="diagram-details reverse">
-              <div class="diagram-detail">
+            <div class="diagram-detail -reverse">
+              <div class="diagram-detail__card">
                 <!--transparent borders-->
                 <div class="icon -borders"></div>
                 <!--background-color-->
                 <div class="icon -color">
                   <!--img-->
-                  <img class="resize" src="./public/img/icon-behavioral.png">
+                  <img class="resize" src="./public/img/icon-behavioral.png" alt="">
                 </div>
-                <p class="text -small -color-accent -caps -bold centered">BEHAVIORAL MODELING</p>
-                <p class="text -small content">Florence can generate many different messages which, unchecked, could make her a little too friendly. A system that learns the preferences of users will be helpful in tailoring which kinds of messages are sent, when, and in what frequency.</p>
+                <h4 class="text--subheading">BEHAVIORAL MODELING</h4>
```

Comment:
Can this be in not in all caps and use the `-caps` modifier class or `text-transform: uppercase` to `.text--subheading`?